packages/account/src/Sections/Profile/PhoneVerification/__test__/verification-link-expired-modal.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import VerificationLinkExpiredModal from '../verification-link-expired-modal';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+const mock_back_router = jest.fn();
+jest.mock('react-router', () => ({
+    ...jest.requireActual('react-router'),
+    useHistory: () => ({
+        goBack: mock_back_router,
+    }),
+}));
+
+describe('VerificationLinkExpiredModal', () => {
+    let modal_root_el: HTMLElement;
+    const mockSetShowVerificationLinkExpiredModal = jest.fn();
+
+    beforeEach(() => {
+        mockSetShowVerificationLinkExpiredModal.mockClear();
+        mock_back_router.mockClear();
+    });
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    const mock_store = mockStore({});
+
+    const buttons = [/Send new link/, /Cancel/];
+
+    const renderComponent = () => {
+        render(
+            <StoreProvider store={mock_store}>
+                <VerificationLinkExpiredModal
+                    should_show_verification_link_expired_modal
+                    setShouldShowVerificationLinkExpiredModal={mockSetShowVerificationLinkExpiredModal}
+                />
+            </StoreProvider>
+        );
+    };
+
+    it('it should render VerificationLinkExpiredModal', () => {
```

Comment:
```suggestion
    it('should render VerificationLinkExpiredModal', () => {
```

Reply:
It should be like this in all other tests as well..

Reply:
oops will change it 😁